src/jit/lsra.h

```diff
@@ -42,11 +42,21 @@ regMaskTP calleeSaveRegs(RegisterType rt)
 struct LocationInfo
 {
     LsraLocation  loc;
-    Interval    * interval;
-    GenTree     * treeNode;
-
-    LocationInfo(LsraLocation l, Interval *i, GenTree *t)
-    : loc(l), interval(i), treeNode(t) {}
+    Interval*     interval;
+    GenTree*      treeNode;
+
+    // Reg Index in case of multi-reg result producing call node.
+    // Indicates the position of the register that this location refers to.
+    // The max bits needed is based on max value of MAX_RET_REG_COUNT value
+    // across all targets and that happens 4 on on Arm.  Hence index value
+    // would be 0..MAX_RET_REG_COUNT-1. 
+    unsigned      multiRegIdx : 2;  
```

Comment:
It may not matter (I presume that the C++ compiler will reorganize our structs in ret builds), but I would put this above the pointers, since on a 64-bit system it will fit in the space between `loc` and `interval`